sdk/storage/azure-storage-blob/tests/_shared/testcase.py

```diff
@@ -140,6 +148,18 @@ def account_url(self, name, storage_type):
         :param str name: Storage account name
         :param str storage_type: The Storage type part of the URL. Should be "blob", or "queue", etc.
         """
+        if "StorageAccount" in str(type(name)):
```

Comment:
>name [](start = 40, length = 4)

Probably we want to rename "name" to "storage_account"

Reply:
Fixed